packages/web-components/src/components/va-file-input-multiple/test/va-file-input-multiple.spec.ts

```diff
@@ -1,17 +1,21 @@
 import { VaFileInputMultiple } from '../va-file-input-multiple';
 import { FileDetails } from "../FileDetails";
+import { FileIndex } from '../FileIndex';
 
 describe('checking buildFilesArray output', ()=> {
-  const fileArray = [new File(["foo"], "foo.txt"), new File(["bar"], "bar.txt")];
+  const fileArray:FileIndex[] = [
+    {file:new File(["foo"], "foo.txt"), key: 0, content:null},
+    {file:new File(["bar"], "bar.txt"), key: 1, content:null}
+  ];
   it('should mark the first file as changed', async () => {
     const component = new VaFileInputMultiple();
     let filesArray = component.buildFilesArray(fileArray, false, 0);
     let expectedOutput:FileDetails[] = [ {
-      file: fileArray[0],
+      file: fileArray[0].file,
```

Comment:
Will this expected output change affect any of the existing vets-website implementations?